### PR TITLE
getting variable_id

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/rest-api/applications.md [hunk 1]
```diff
@@ -286,6 +286,19 @@ To add binary-based files (e.g. images), they need to be [`base64 encoded`](../v
 }
 ```
 
+### Getting variable_id
+
+#### Example
+
+```
+curl -XGET -H "X-Auth-Token: $CM_API_KEY" -H "Content-type: application/json" "https://api.codemagic.io/apps/$YOUR_APP_ID/variables"
```
Comment (nevercode-kevin) [inline content/rest-api/applications.md:294]:
```suggestion
curl -XGET -H "X-Auth-Token: $CM_API_KEY" -H "Content-type: application/json" "https://api.codemagic.io/apps/YOUR_APP_ID/variables"

```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by icarusdust** [comment]
Done

**Review by nevercode-kevin — APPROVED** [review]
LGTM

File: content/rest-api/applications.md [hunk 2]
```diff
@@ -286,6 +286,19 @@ To add binary-based files (e.g. images), they need to be [`base64 encoded`](../v
 }
 ```
 
+### Getting variable_id
+
+#### Example
+
+```
+curl -XGET -H "X-Auth-Token: $CM_API_KEY" -H "Content-type: application/json" "https://api.codemagic.io/apps/YOUR_APP_ID/variables"
```
Comment (VeArnold) [inline content/rest-api/applications.md:294]: still looks like YOUR_APP_ID not $YOUR_APP_ID

File: content/rest-api/applications.md [hunk 3]
```diff
@@ -286,6 +286,19 @@ To add binary-based files (e.g. images), they need to be [`base64 encoded`](../v
 }
 ```
 
+### Getting variable_id
```
Comment (VeArnold) [inline content/rest-api/applications.md:289]: ### Getting variable ID

perhaps

**Comment by VeArnold** [comment]
I think we should also add a reference about it to the places where it is required.

**Pull request merged**